docs/Displaying a User Interface/Template Images/index.md

```diff
@@ -8,6 +8,19 @@ You can easily display text, images, and buttons using the @![iOS]`SDLScreenMana
 | secondaryGraphic | The second image in a template that supports multiple images |
 
 ## Showing Images
+### SDLArtwork
+Images should be of type SDLArtwork
+
+@![iOS]
+```swift 
+
+let <#your variable name here#> = SDLArtwork(image: UIImage(named: <#Name to Upload As#>)!, persistent: true, as: .PNG /* or .JPG */)
+
+```
+!@ 
```

Comment:
1. You need to also include Objective-C code (see other code blocks for how that's supposed to happen) and stubs for other languages so that Java Suite and JavaScript engineers can fill in the blanks.
2. Please also remove the extra spacing lines
3. Provide an example for uploading as a template image.
4. The "Name to Upload As" is not correct. The name there will not be used in the initializer you're showing. The image name will be created from an md5 hash of the image data. If they want to specify a name, there would be a different initializer (which you should probably also show.